### ISSUE TITLE
Kirgo grants double Experience in the arena when defeated before

### ISSUE THREAD

**Issue by AmProsius** [AmProsius/gothic-1-community-patch#206]
**Describe the bug**
The player character gets Experience again when defeating Kirgo in the arena, if he defeated Kirgo before.

**Expected behavior**
The player character doesn't get Experience again when defeating Kirgo in the arena, if he defeated Kirgo before.

**Steps to reproduce the issue**
1. Defeat Kirgo.
2. Challenge Kirgo and defeat him again in the arena.

**Comment by szapp** [comment]
Shouldn't this be solved already by #12?

**Comment by AmProsius** [comment]
I don't think so. There are neither ranged weapons nor killing involved in this issue. But I also haven't confirmed it yet.
